src/base/sys_info.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SRC_BASE_SYS_INFO_H_
+#define SRC_BASE_SYS_INFO_H_
+
+#include "absl/strings/ascii.h"
+#include "absl/strings/match.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/string_view.h"
+#include "base/status.h"
+
+namespace openmldb::base {
+
+constexpr const char* MEM_TOTAL = "MemTotal";
+constexpr const char* MEM_BUFFERS = "Buffers";
+constexpr const char* MEM_CACHED = "Cached";
+constexpr const char* MEM_FREE = "MemFree";
+constexpr const char* SLAB = "Slab";
```

Comment:
man free
> cache  Memory used by the page cache and slabs (Cached and SReclaimable in /proc/meminfo)

Reply:
@dl239  I think you should use `SReclaimable` instead of `Slab`

Reply:
I see the manual of `free` is below under centos 7.5. centos 7.7 shows as you quote.
> cache  Memory used by the page cache and slabs (Cached and Slab in /proc/meminfo
in fact, the computation of `MemAvailable` is more complicated, refer [`si_mem_available`](https://elixir.bootlin.com/linux/v6.6/source/mm/show_mem.c).
we can use `SReclaimable`  as a rough estimate. refer [proc docs](https://www.kernel.org/doc/Documentation/filesystems/proc.txt)
> MemAvailable: An estimate of how much memory is available for starting new
              applications, without swapping. Calculated from MemFree,
              SReclaimable, the size of the file LRU lists, and the low
              watermarks in each zone.
              The estimate takes into account that the system needs some
              page cache to function well, and that not all reclaimable
              slab will be reclaimable, due to items being in use. The
              impact of those factors will vary from system to system.